addons/binding/org.openhab.binding.feed/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="feed" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- DEFINITIONS of terms used in the binding:
+	    Feed is a XML document used for providing users with frequently updated content. It is located at URL. The most popular feed formats are RSS and Atom. 
+		Entry is a single element in the Feed, that contains reference (link), short description and other information like images, 
+		comments and etc. Entry in this binding is abstraction for RSS item element and Atom entry element. -->
+
+	<!-- Feed Thing Type -->
+	<thing-type id="feed">
+		<label>Feed</label>
+		<description>Provides the content of the Feed in XML format.</description>
+
+		<channels>
+			<channel id="feed-channel" typeId="feed-channel" />
+		</channels>
+
+		<config-description>
+			<parameter name="URL" type="text" required="true">
+				<label>Feed URL</label>
+				<description>Feed URL used to localize the Feed.</description>
```

Comment:
why "to localize"? Isn't that THE feed url to also get all content from as well?